mdpow/workflows/registry.py

```diff
@@ -0,0 +1,35 @@
+# MDPOW: registry.py
+# 2023 Cade Duckworth
+
+"""
+:mod:`mdpow.workflows.registry` --- Registry of currently supported automated workflows
+=======================================================================================
+
+Each entry in :mod:`mdpow.workflows.registry` corresponds to an :class:`~mdpow.analysis.ensemble.EnsembleAnalysis` for which exists a corresponding
+automated workflow.
```

Comment:
Here should be a list of workflows, essentially manually maintain what's in registry.